src/GitPrompt.ps1

```diff
@@ -330,6 +330,10 @@ $PoshGitVcsPrompt = {
         if ($s) {
             Write-Prompt $s.BeforeText -BackgroundColor $s.BeforeBackgroundColor -ForegroundColor $s.BeforeForegroundColor
             Write-Prompt "Error: $_" -BackgroundColor $s.ErrorBackgroundColor -ForegroundColor $s.ErrorForegroundColor
+            if ($s.Debug) {
+                Write-Host
+                Write-Verbose $_.ScriptStackTrace -Verbose
```

Comment:
👍 consolidated into a single `Write-Verbose` for no particular reason.

> But note that this will not capture any errors in the built-in prompt function as that function calls this one.

True. I'm content with this for 0.x; the prompt function changes substantially in 1.0 anyway.

Though it occurs to me that this whole change isn't all that useful as folks can just look at `$Error` to see what went wrong in either case. 🤷‍♂️